plugin/completion/base_complete.py

```diff
@@ -120,16 +120,20 @@ def init(self, view, settings):
 
         """
         if not view:
+            log.critical(" no view to initialize completer!")
```

Comment:
We are in a concurrent world. It can happen. Very unlikely, but theoretically it is possible.